ggml.c

```diff
@@ -35,12 +35,21 @@
 #include <windows.h>
 #endif
 
+// if C11 or above use stdatomic.h
+#if __STDC_VERSION__ >= 201112L
+#include <stdatomic.h>
```

Comment:
this doesn't work for me on VS2022. I got compile error:
  C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.35.32215\include\vcruntime_c11_stdatomic.h(36,24): error C2061: syntax error: identifier 'atomic_
bool'

Reply:
Interesting... I also use VS2022. I'll check this out.